packages/client/src/containers/CollectionsPage/CollectionsPage.Container.js

```diff
@@ -4,6 +4,9 @@ import './CollectionsPage.style.css';
 import DropDownView from '../../components/CategoriesListDropDown/CategoriesListDropDown.component';
 import SearchInput from '../../components/SearchInput/SearchInput.component';
 import { apiURL } from '../../apiURL';
+import { getLocalStorage, setLocalStorage } from '../../utils/storageHelpers';
+
+const favoriteProductsStorageKey = 'favorite_products';
```

Comment:
You could make it more clear that this is a constant by making it CONSTANT_CASE.